apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -22,11 +22,52 @@ import {
  */
 export class CommandLineConfiguration {
   private static _jsonSchema: JsonSchema = JsonSchema.fromFile(
-    path.join(__dirname, '../schemas/command-line.schema.json'));
+    path.join(__dirname, '../schemas/command-line.schema.json')
+  );
 
   public readonly commands: CommandJson[] = [];
   public readonly parameters: ParameterJson[] = [];
 
+  public static readonly defaultBuildCommandJson: CommandJson = {
+    commandKind: RushConstants.bulkCommandKind,
+    name: RushConstants.buildCommandName,
+    summary: '(EXPERIMENTAL) Build all projects that haven\'t been built, or have changed since they were last'
```

Comment:
Let's delete `(EXPERIMENTAL)`.  The limitation of `package-deps.json` that made incremental builds unreliable was fixed a while ago.